ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -54,7 +54,17 @@
   "resultsCount" : $matches.size(),
   "results" : [
 #foreach ($p in $matches)
-  $p.toJSON()#if ($foreach.hasNext()),
+   #set ($match = $p.toJSON())
```

Comment:
Wouldn't it be simpler to do this [in java](https://github.com/phenotips/patient-network/blob/master/similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientGenotypeSimilarityView.java#L255)?